src/features/formData/submit/submitFormDataSagas.ts

```diff
@@ -330,7 +330,7 @@ export function* postStatelessData({ field, componentId }: SaveDataParams) {
         // If the saga were cancelled (takeLatest), we would abort the HTTP request/promise
         // to ensure we do not update the redux-state with staled data.
         abortController.abort();
-        console.warn('Request aborted due to saga cancellation');
+        window.logWarn('Request aborted due to saga cancellation');
```

Comment:
I think this one should be an 'info' level log item, not a warning. Typing too fast should not be a cause for concern! 😁